Hobbies.java

```diff
@@ -0,0 +1,32 @@
+package org.example.homework;
+
+public class Hobbies {
+    private final String hobby;
+    private final int frequency;
+    private final String listOfAdresses;
```

Comment:
I'd rather keep it as 
```suggestion
    private final List<String> listOfAdresses;
```
it would be easier to manipulate in case of multiple addresses